curriculum/challenges/english/25-front-end-development/quiz-dom-manipulation-and-click-event-with-javascript/66edd07682767adff3a6231e.md

```diff
@@ -91,258 +91,225 @@ document.querySelectorAll("a:not([href])");
 
 ---
 
-```js
-const target = document.querySelector("p#target");
-target.replaceChild("That's it", target.lastChild);
-```
+It adds text to all arrays and objects in your JavaScript file.
 
 #### --answer--
 
-```js
-const target = document.getElementById("target");
-target.removeChild(target.lastChild);
-target.append("That's it.");
-```
+It returns the plain text content of an element.
 
 ### --question--
 
 #### --text--
 
-Which one of these affirmations is false?
+What does the `innerHTML` property do?
 
 #### --distractors--
 
-The `innerHTML` element's property returns all the HTML content inside it as text.
+It is a property of the `Element` used to get all text content for an HTML element.
 
 ---
 
-The `innerText` element's property only returns the text content that is shown on screen.
+It is a property of the `Element` used to add annotations to your HTML.
 
 ---
 
-The `textContent` node's property returns all the text content inside it.
+It is a property of the `Element` used to trigger events in your HTML. 
 
 #### --answer--
 
-The `htmlContent` node's property only returns HTML content that is shown on screen.
+It is a property of the `Element` used to set or update parts of the HTML markup.
 
 ### --question--
 
 #### --text--
 
-Which of the following JavaScript lines do not insert a thematic break at the end of the document body?
+Which of the following is the correct way to use the `addEventListener()` method?
 
 #### --distractors--
 
 ```js
-document.body.insertBefore(document.createElement("hr"), null);
+btn.addEventListener("accept", () => alert("You clicked the button"));
 ```
 
 ---
 
 ```js
-document.body.appendChild(document.createElement("hr"));
+btn.addEventListener("allow", () => alert("You clicked the button"));
 ```
 
 ---
 
 ```js
-document.body.after(document.createElement("hr"));
+btn.addEventListener("trigger", () => alert("You clicked the button"));
 ```
 
 #### --answer--
 
 ```js
-document.body.append("<hr/>");
+btn.addEventListener("click", () => alert("You clicked the button"));
 ```
 
 ### --question--
 
 #### --text--
 
-Which of the following statements is false?
+Which of the following is NOT a valid method?
 
 #### --distractors--
 
-`window` is an alias for `navigator.window`:
-
-```js
-window === navigator.window // returns true
-```
+`getElementById()`
 
 ---
 
-You can use the user screen properties (like position and size) using the `Window` object.
+`querySelector()`
 
 ---
 
-`navigation` is an alias for `window.navigation`:
-
-```js
-navigation === window.navigation // returns true
-```
+`createElement()` 
 
 #### --answer--
 
-You can access the clipboard using:
-
-```js
-await browser.clipboard.read();
-```
+`addElementToDOM()`
 
 ### --question--
 
 #### --text--
 
-Assuming the JavaScript variable `element` is an HTML `Element` object, which code does not do the same thing as the others?
+Which of the following is the correct way to work with the `setAttribute()` method?
```

Comment:
```suggestion
Which of the following will set the `class` attribute to `my-class`?
```

Technically they are *all* correct, they just won't do the thing we likely wanted them to do.